MapboxDirections/MBRouteOptions.swift

```diff
@@ -111,6 +118,35 @@ open class RouteOptions: DirectionsOptions {
      Currently, you can only specify a single road class to avoid.
      */
     @objc open var roadClassesToAvoid: RoadClasses = []
+    
+    /**
+     A bias which determines whether the route should prefer or avoid the use of alleys. The
+     allowed range of values is from -1.0 to 1.0, where -1 indicates preference to avoid
+     alleys, 1 indicates preference to favor alleys, and 0 indicates no preference.
+     
+     Defaults to 0.
+     This property has no effect unless profile identifier is set to `MBDirectionsProfileIdentifier.walking`.
+     */
+    @objc open var alleyBias: Double = 0
+    
+    /**
+     A bias which determines whether the route should prefer or avoid the use of roads or paths
+     that are set aside for pedestrian-only use (walkways). The allowed range of values is from
+     -1.0 to 1.0, where -1 indicates indicates preference to avoid walkways, 1 indicates preference
+     to favor walkways, and 0 indicates no preference.
+     
+     Defeaults to 0.
+     This property has no effect unless profile identifier is set to `MBDirectionsProfileIdentifier.walking`.
+     */
+    @objc open var walkwayBias: Double = 0
+    
+    /**
+     Walking speed in meters per second. Must be between 0.14 and 6.94 meters per second.
+     
+     Defaults to 1.42
+     This property has no effect unless profile identifier is set to `MBDirectionsProfileIdentifier.walking`.
+     */
+    @objc open var walkingSpeed: CLLocationSpeed = 1.42
```

Comment:
Should we leave open the possibility that the cycling profile may support specifying a maximum attainable speed in the future? Naming the option this way means we’d later have to add a `cyclingSpeed` property, and neither would be applicable in the driving profiles. We should avoid a situation where DirectionsOptions or RouteOptions is saddled with too many options that are mutually exclusive. We could name the property `speed` or `expectedSpeed`, still with the note that it’s ignored for profiles other than walking.